fiftyone/utils/transformers.py

```diff
@@ -21,13 +21,17 @@
 from fiftyone.zoo.models import HasZooModel
 import fiftyone.utils.torch as fout
 
+
 fou.ensure_torch()
 import torch
 
+from torchvision.transforms import functional as F
```

Comment:
Since torchvision isn't  a requirement, I don't think you can import it like this